pallas-network/src/miniprotocols/localstate/queries_v16/mod.rs

```diff
@@ -201,19 +200,21 @@ pub struct StakeDistribution {
 #[derive(Debug, Encode, Decode, PartialEq, Clone)]
 pub struct Pool {
     #[n(0)]
-    pub stakes: Fraction,
+    pub stakes: RationalNumber,
 
     #[n(1)]
     pub hashes: Bytes,
 }
 
+/// Type used at [GenesisConfig], which is a fraction that is CBOR-encoded
+/// as an untagged array.
 #[derive(Debug, Encode, Decode, PartialEq, Clone)]
 pub struct Fraction {
     #[n(0)]
     pub num: u64,
 
     #[n(1)]
-    pub dem: u64,
+    pub den: u64,
 }
```

Comment:
Do we need this struct now that we're using `RationalNumber` instead?

Reply:
FYI Reply is below